packages/office-addin-manifest/src/manifestHandler/manifestHandlerJson.ts

```diff
@@ -1,29 +1,61 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+import { TeamsAppManifest } from "@microsoft/teamsfx-api";
```

Comment:
So the TeamsAppManifest interface is not part of the new teams-manifest pacakge?

Reply:
Fixed